util/keyfile/src/config.rs

```diff
@@ -1,48 +1,69 @@
 // Copyright (c) 2018-2021 The MobileCoin Foundation
 
-use rand::{rngs::StdRng, SeedableRng};
-use std::path::PathBuf;
+use std::{cmp, fs, path::PathBuf};
+
 use structopt::StructOpt;
 
+// Hack to work around Vec special handling in structopt
+type VecBytes = Vec<u8>;
 /// Configuration for generating key files for a new user identity
 #[derive(Debug, StructOpt)]
 pub struct Config {
-    /// Optional FogURL for the accounts
-    #[structopt(short, long)]
-    pub acct: Option<String>,
+    /// Fog Report URL
+    #[structopt(long)]
+    pub fog_report_url: Option<String>,
```

Comment:
Optional: Use `mc_util_uri::FogUri` to validate the URL format

Reply:
I'll do this separately if it's still desired.